fobi_custom/override_theme/templates/override_theme/form_entry_submitted_ajax.html

```diff
@@ -1,9 +1,10 @@
 <div class="col col-lg-6 card">
   <div class="card-body">
-    <h2 class="card-title">Survey submitted!</h2>
+    <h2 class="card-title">{{form_entry}} submitted!</h2>
```

Comment:
I like that we're displaying the title of the survey now, but I found it a bit hard to parse with a long survey name:

![Screen Shot 2019-06-11 at 5 24 57 PM](https://user-images.githubusercontent.com/14170650/59360235-5b691400-8cf5-11e9-9e36-d6a635dad46c.png)

What about something like:

> Survey _"Test saving ACS data"_ submitted!

Reply:
Good idea!